lms/djangoapps/certificates/management/commands/cert_allowlist_generation.py

```diff
@@ -0,0 +1,79 @@
+"""
+Management command to generate allowlist certificates for one or more users in a given course run.
+"""
+
+import logging
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand, CommandError
+from opaque_keys.edx.keys import CourseKey
+
+from lms.djangoapps.certificates.generation_handler import generate_allowlist_certificate_task
+
+User = get_user_model()
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Management command to generate allowlist certificates for one or more users in a given course run.
+
+    Example usage:
+    ./manage.py lms cert_allowlist_generation -u edx verified -c course-v1:edX+DemoX+Demo_Course
+    """
+
+    help = """
+    Generate allowlist certificates for one or more users in a given course run.
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-u', '--user',
+            nargs="+",
+            metavar='USER',
+            dest='user',
+            required=True,
+            help='user or comma-separated list of users for whom to generate allowlist certificates'
```

Comment:
I believe that "comma-separate" is actually "space-separated" list. Django might do something fancy, but I think it just uses `argparse`.

Test code:
```
import argparse
parser = argparse.ArgumentParser()
parser.add_argument(
    '-u', '--user',
    nargs="+",
    metavar='USER',
    dest='user',
    required=True,
    help='user or comma-separated list of users for whom to generate allowlist certificates'
)
args = parser.parse_args()
print(args)
```
Test result:
```
$ python3 test_args.py --user matt,christie
> Namespace(user=['matt,christie'])
$ python3 test_args.py --user matt, christie
> Namespace(user=['matt,', 'christie'])
$ python3 test_args.py --user matt christie
> Namespace(user=['matt', 'christie'])
```